### ISSUE TITLE
Ensure targeting pack dependency versions are pinned for patch builds.

### ISSUE THREAD

**Issue by JunTaoLuo** [aspnet/AspNetCore#8633]
https://github.com/aspnet/AspNetCore/pull/8490#issuecomment-473559281

**Comment by Eilon** [comment]
Needs to be done at ~RTM time, before we ship any patch. Might need to be done earlier if we do a patch prototype prior to RTM, which is a likely possibility.

**Comment by natemcmaster** [comment]
@Pilchie this one affects servicing 3.0.x and would be good to review for RTM.

**Comment by anurse** [comment]
If I'm understanding correctly, the main work here is to ensure the `deb` and `rpm` packages have a *fixed* dependency to `3.0.0` of the .NET Core targeting pack, right? Seems straightforward enough. We can't actually make that change until we ship though, since we *do* need to reference the active preview version as we're still shipping previews.

**Comment by dougbu** [comment]
At this point, our incoming ref packages (from Extensions and core-setup) should both be pinned at 3.0.0. The remaining work before we close this issue is to stop building the AspNetCore targeting pack after 3.0.1 ships.

@JunTaoLuo please confirm

**Comment by JunTaoLuo** [comment]
That is correct, we have pinned the ref pack version in the deb and rpm installers. However, there shouldn't be any work required to stop building the targeting pack since it's already conditional on 3.0.1: https://github.com/aspnet/AspNetCore/blob/release/3.0/Directory.Build.props#L93-L94.

**Comment by JunTaoLuo** [comment]
Curious if any of this logic will be impacted by the 3.0.2 changes you're making @dougbu ?

**Comment by dougbu** [comment]
Not really. I am however concerned about what will happen w/ the checked-in approach once we stop building the targeting pack. Will the infrastructure still build ref/ projects and reference them? Can we test today or tomorrow to make sure we're :ok:?

**Comment by JunTaoLuo** [comment]
I'll try it out and report back.

**Comment by wtgodbe** [comment]
@JunTaoLuo are you on this for 3.1.1 as well?

**Comment by JunTaoLuo** [comment]
I think there's no longer any action to take here. We have updated the targeting pack dependencies to use the version of Microsoft.NETCore.App.Ref dependency: https://github.com/aspnet/AspNetCore/blob/release/3.0/src/Installers/Debian/TargetingPack/debian_config.json.in#L33. The same is also true for 3.1 and master. Given that in 3.0 we have pinned the dependency at 3.0.0 (and 3.1.0 in 3.1) the dependency version will be correct if we ever need to service the targeting pack again.
